docs/aws-ec2/README.md

```diff
@@ -0,0 +1,22 @@
+# EC2 Developer Setup
+
+## Background
+
+For easy development purposes, it is beneficial to setup a ec2 instance for development as the memory requirements for armada can be quite intense.  For example, a mac M1 with 8 gb is too small for development.  Some images will also not be supported on a M1 so it is beneficial to setup a ec2 instance for development.
+
+## Instructions
+
+- We suggest a t3.xlarge instance from aws ec2 with Amazon Linux.  16 GB of memory is suggested.
+- During selection of instance, Add a large volume to your ec2 instance (100 gb)
```

Comment:
I find 30 GB is a reasonable minimum fwiw

Reply:
For memory or storage?

Reply:
Storage, but it's fine to shoot high